src/ensembl/.eslintrc.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
     'react/prop-types': 0,
     'react/no-unescaped-entities': 0,
     'react-hooks/rules-of-hooks': 2,
-    'prettier/prettier': 0
+    'prettier/prettier': 0,
+    'no-unused-vars': ['warn', { args: 'after-used' }]
```

Comment:
To prevent the linter from complaining when arguments before a used argument are unused.

Example:

```
function(foo, bar, baz) {
    return baz;
}
```
in the example above, linter will not complain of the variables foo and bar.